Scheduler/observing_blocks_maintel/BLOCK-T87.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "BLOCK-T87",
+  "id": "d41972b5-4c3d-43e8-9f90-b9bc29f7d037",
+  "program": "BLOCK-T87",
+  "constraints": [],
+  "scripts": [
+    {
+      "name": "maintel/offset_camera_hexapod.py",
+      "standard": true,
+      "parameters": {
+        "z": -1500
```

Comment:
I think for this to work at the summit with the system in its current state, you might have to ignore all the other TCS components and leave just the camera hexapod out.